assets/js/components/HostsList.jsx

```diff
@@ -86,16 +110,30 @@ const HostsList = () => {
         key: 'cluster',
         className: 'w-40',
         render: (cluster) => {
-          return cluster?.name;
+          return <ClusterLink cluster={cluster}>{cluster?.name}</ClusterLink>;
         },
       },
       {
         title: 'SID',
         key: 'sid',
-        filter: true,
-        render: (content, { cluster }) => (
-          <ClusterLink cluster={cluster}>{content}</ClusterLink>
-        ),
+        filter: (filter, key) => (element) =>
+          element[key].some((sid) => filter.includes(sid)),
+        render: (sids, { sap_systems }) => {
+          let sidsArray = sids.map((sid, index) => [
```

Comment:
Maybe here, instead of mapping the `sids` element, we could map the `sap_systems` list, which already has the sid in each element. This way, we could space the `getSAPInstanceFromSID` function.

I mean, we already have all the information in the `sap_systems`. It is nice that we can send the list of sids as strings, so we don't need to do hacky things in the filters